puppet/manifests/site.pp

```diff
@@ -1,4 +1,4 @@
-node default {
+node /.*\.theforeman\.org$/ {
```

Comment:
> Important: Make sure all of your node definition name regexes match non-overlapping sets of node names. If a node’s name matches more than one regex, Puppet makes no guarantee about which matching definition it will get.

https://www.puppet.com/docs/puppet/7/lang_node_definitions.html

:crying_cat_face: